frontend/server/controllers/RunController.php

```diff
@@ -335,7 +335,7 @@ public static function apiCreate(Request $r) {
             }
 
             SubmissionLogDAO::create(new SubmissionLog([
-                'user_id' => $r->user->user_id,
+                'user_id' => $r->user->user_id ?? null,
```

Comment:
```suggestion
                'user_id' => !is_null($r->user) ? $r->user->user_id : null,
```

nota que no se puede usar `??` aquí porque para evaluar el lado izquierdo de ese operador, tuviste que haber accedido a `$r->user`, que _puede_ ser nulo!

y recuerda que en general `$x ?? null` hay que reemplazarlo por `$x` porque el `?? null` es superfluo.

Reply:
Listo